types/kendo-ui/index.d.ts

```diff
@@ -3748,7 +3748,7 @@ declare namespace kendo.ui {
         aggregates?: any;
         attributes?: any;
         columns?: any;
-        command?: GridColumnCommandItem[];
+        command?: string|(string|GridColumnCommandItem)[];
```

Comment:
Could you please add a test case, showing intended usage, that would fail without your changes? This will help me understand your intention.